mlfinlab/bet_sizing/targeting.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains functionality for determining bet sizes for investments based on volatility targeting.
+"""
+
+import numpy as np
+
+
+def volatility_targeting(returns: np.ndarray, target_vol: float, annualize_factor: int):
+    """
+    Calculate the position size to make the expected volatility of the portfolio be the target variability.
+
+    :param returns: (np.ndarray) This is series of returns of your strategy or a asset.
```

Comment:
It seems like this function also can work with `pd.DataFrame` and `pd.Series` as `returns` parameter. Is this function expected to take multiple series as an input?

Reply:
Not it is not. I should clarify it.